data/category-analytics

```diff
@@ -0,0 +1,30 @@
+# List of all domains being used in log collection or monitor.
+
+include:category-ads
+
+aax-us-east.amazon-adsystem.com
+alb.reddit.com
+analytics.brave.com
+analytics.map.qq.com
+analytics.plex.tv
+app-measurement.com
+be-nl.norton.com
+c.s-microsoft.com
```

Comment:
If providing the referring sources turns to be difficult and painful, you could just concatenate all the domain lists that you have accessed during doing this PR, then give me the all-in-one list for me to do the review.